src/messenger-main.tsx

```diff
@@ -5,13 +5,14 @@ import { connectContainer } from './store/redux-container';
 import { Main } from './Main';
 import { ZUIProvider } from '@zero-tech/zui/ZUIProvider';
 import { Provider as AuthenticationContextProvider } from './components/authentication/context';
-import { setActiveConversationId } from './store/chat';
+import { setActiveConversationId, validateAndSetActiveConversationId } from './store/chat';
 
 export interface Properties {
   isAuthenticated: boolean;
 
   match: { params: { conversationId: string } };
   setActiveConversationId: (id: string) => void;
```

Comment:
We can remove this now?

Reply:
it's being used with `componentDidUpdate`

Reply:
Pretty sure we'd want the same validation when the component updates. Otherwise if we use a linkable channel in the chat somehow (think "#zos-core") then someone could click a conversation they aren't a part of.